README.md

```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
 ![Application-Architecture-Diagram](./docs/images/application-architecture-diagram/application_architecture_diagram.png)
 
 ## User Stories 
+We constructed user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying customers' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+### Initial Market Research
+1. As a busy parent, I want search functionality with a category filtering option to save time searching for the right product.
+
+2. As a busy parent, I want a contact form 24/7 to quickly ask questions or queries about products and services in the online shop, regardless of time or location.
+
+3. As a user who often changes residence, I need to easily update my profile to ensure accurate delivery despite frequent changes in residence.
+
+4. As a user, I want the convenience of a Product Search Bar that allows me to find specific pantry items effortlessly. This will save me time and ensure that I quickly locate the products I'm looking for.
+
+5. As a user who loves new discoveries, I want a new arrivals section to give me new suggestions on Japanese products I might enjoy. 
+
+6. As a cooking enthusiast, I want to have an intuitive platform where I can enjoy browsing the different ingredients and products exclusive to Japan.
+
+7. As a non-Japanese user who needs help understanding the Japanese language, I want a website with the products translated into English to understand the product details better.
+
+8. As an administrator, I want the ability to efficiently manage users by viewing their information and assisting with account-related matters.
+
+9. As an administrator, the ability to perform CRUD operations on products and select quantities will enable me to manage product inventory efficiently and maintain accurate stock listings.
+
```

Comment:
Can you add one more user story about the Data Summary for admin dashboard, a summary view of total users, total products, total orders and total revenue?